sklearn/svm/tests/test_svm.py

```diff
@@ -275,9 +275,9 @@ def test_tweak_params():
     """
     clf = svm.SVC(kernel='linear', C=1.0)
     clf.fit(X, Y)
-    assert_array_equal(clf.dual_coef_, [[.25, -.25]])
+    assert_array_equal(clf.dual_coef_, [[-.25, .25]])
     assert_array_equal(clf.predict([[-.1, -.1]]), [1])
-    clf.dual_coef_ = np.array([[.0, 1.]])
+    clf._dual_coef_ = np.array([[.0, 1.]])
```

Comment:
Note this. Since this PR fixes sign of `dual_coef_` in the same way `intercept_` is fixed (which is done by introducing another property `_intercept_`), then it'd be a bit non-intuitive to set new dual coefficients, since a user would need to change `_dual_coef_` instead of `dual_coef_` (Should I document it somewhere?).

Though, this usecase seems very odd and rare to me in the first place, I can't imagine a case when I'd need to change dual coefficients.